cfgov/apache/conf.d/redirects.conf

```diff
@@ -252,9 +252,13 @@ RedirectMatch permanent (.*)/realestateprofessionals(.*)  $1/real-estate-profess
 RedirectMatch permanent ^/real-estate[/]?$  /know-before-you-owe/real-estate-professionals/
 
 # Retirement redirects
-RedirectMatch permanent (?i)^/retirement/claiming-social-security/?$ /retirement/before-you-claim/
-RedirectMatch permanent (?i)^/jubilacion/?$  /retirement/before-you-claim/es/
-RedirectMatch permanent (?i)^/retirement/?$  /retirement/before-you-claim/
+RedirectMatch permanent (?i)^/retirement/claiming-social-security/?$ /consumer-tools/retirement/before-you-claim/
+RedirectMatch permanent (?i)^/jubilacion/?$ /es/herramientas-del-consumidor/jubilacion/antes-de-solicitar/
+RedirectMatch permanent (?i)^/retirement/?$  /consumer-tools/retirement/before-you-claim/
+
+# Redirect from /es/ suffix urls
+RedirectMatch permanent (?i)^/consumer-tools/financial-well-being/es/ /es/herramientas-del-consumidor/bienestar-financiero/
```

Comment:
Is this financial-well-being entry supposed to be here, or is it that this just hasn't been rebased with main?

Reply:
Hah must have double pasted (check lines 263, 264)

Reply:
Fixed in e291951d34